.travis.yml

```diff
@@ -16,11 +16,12 @@ addons:
       - python
       - python-pip
       - python2.7-dev
+      - clang-format-3.9
```

Comment:
In paddle repo and others, we use clang-format-3.8
https://github.com/PaddlePaddle/Paddle/blob/develop/.travis.yml#L29

Reply:
This is specified by already merged DS2 c++ codes.